pkgs/development/ocaml-modules/bitstring/ppx.nix

```diff
@@ -3,6 +3,10 @@
 , ounit
 }:
 
+if !lib.versionAtLeast ppxlib.version "0.22.0"
+then throw "ppx_bitstring is not available for OCaml ${ocaml.version}"
```

Comment:
I think referencing the ppxlib version is more helpful here if someone encounters the error.

Reply:
I am not sure if ocaml has different policies but I think such throw are not permitted in other areas of nixpkgs.

Reply:
It's completely normal, `disabled` in `buildPythonPackage` does the exact same thing.

Reply:
Fixed the error message.